### ISSUE TITLE
add-noreferrer-quick-fix shouldn't add duplicated rel attribute

### ISSUE THREAD

**Issue by jasonlyu123** [sveltejs/language-tools#1811]
### Describe the bug

add-noreferrer-quic-fix adds another rel attribute when there is already a rel attribute.

### Reproduction

```svelte
<a target="_blank" rel="noopener">test</a>
```

### Expected behaviour

add the value to the existing rel attribute. 

### System Info

- OS: Windows
- IDE: VSCode

### Which package is the issue about?

None

**Comment by cooolbros** [comment]
Yes I can reproduce this, I will see if I can come up with a fix
